documentation/directory-structure.md

```diff
@@ -30,140 +30,102 @@ You may be also interested in following:
 > The structure described in this article is strictly enforced. There exist some exceptions for historical reasons.
 > These exceptions are not allowed going forward.
 
-## `specification` folder
-
-The `specification` folder is the root folder for all service specifications.
+## Key concepts
 
-Each child of the `specification` folder corresponds to a `service` specification for given Azure team. Here we denote such folder as `<azureTeam>`.
-In advanced cases for big teams the `<azureTeam>` folder can host multiple services, known as `service group`.
-Read [the relevant section](#advanced-scenario-service-group) for details.
+The directory structure is a reflection of a few key concepts.
 
-Given `<azureTeam>` has following structure:
+Familiarize yourself with the concepts of `service`, `service group` and `uniform versioning`
+as explained in the [glossary].
 
-- `<azureTeam>/<typeSpecFolder>` (multiple folders)
-- `<azureTeam>/resource-manager`
-- `<azureTeam>/data-plane`
-
-The `<azureTeam>/<typeSpecFolder>` folders contain the TypeSpec specification for the given `service` or `service group`.
-You can find details on the name and contents of these folders in [TypeSpec directory structure].
-You can learn more about TypeSpec at [aka.ms/azsdk/typespec] and [aka.ms/typespec].
-
-## `resource-manager` and `data-plane` folders
+## `specification` folder
 
-The `<azureTeam>/resource-manager` contains the ARM OpenAPI specifications emitted from TypeSpec in `<azureTeam>/<typeSpecFolder>`.
+The `specification` folder is the root folder for all `service` specifications.
 
-The `resource-manager` folder has exactly one child folder whose name matches the **Resource Provider (RP) Namespace** (`<RPNS>`), 
-such as `Microsoft.Automation` (full list of namespaces is [here][Resource Provider list]).
-There is 1 to 1 correspondence between an RP and an RP namespace.
-There must be **exactly one** RP namespace folder under given `resource-manager` folder.
-We denote such folder as `resource-manager/<RPNS>`.
+Each child of the `specification` folder corresponds to `<azureTeam>`. 
+All the `service` (see [glossary]) specifications owned by that team are rooted in their `<azureTeam>` folder.
 
-The `<azureTeam>/data-plane` contains the data-plane OpenAPI specifications emitted from TypeSpec in `<typeSpecFolder>`. 
-The `data-plane` folder has no child `<RPNS>` folder. However, it can have a set of `<dataPlaneSubfolder>` folders.
+Given `<azureTeam>` folder has following structure:
 
-## AutoRest configuration `README.md` files
+- `<azureTeam>/<typeSpecSrc>` (multiple `<TypeSpecSrc>` folders are allowed)
+- `<azureTeam>/resource-manager/<RPNS>` (exactly one `<RPNS>` folder is allowed)
+- `<azureTeam>/data-plane/<namespace>` (multiple `<namespace>` folders are allowed)
```

Comment:
What is a namespace for data-plane? I don't think we have this concept. If we do, then it needs to be in the glossary.

I did some scanning of the repo and I think the "namespace" concept was originally for the .NET SDK namespace which doesn't map to all the languages we support today. The .NET namespace is al cilent-side concept and not a service-side concept (like RPNS). If I'm right, then we need to clean this up too.

Reply:
@JeffreyRichter I proposed to call it `<namespace>` based on all the `data-plane` children folder  names I observed. I provide a full list at the end of this comment.

All the guidance we had so far said that `data-plane` child folders can be arbitrary.

If you don't like the idea of naming it `<namespace>` do you have another proposal? `<serviceGroupName>` ? In case of ARM `resource-manager/<RPNS>` corresponds exactly 1 to 1 to `resource-manager/"service group name"`

Here is the full list of data-plane child folder names, deduplicated:

```
/data-plane/AnomalyDetector
/data-plane/ApiCenter.DataApi
/data-plane/AsyncBatchManagement
/data-plane/AutoSuggest
/data-plane/Azure.AI.ChatProtocol
/data-plane/Azure.AI.DocumentTranslation
/data-plane/Azure.AI.TextTranslation
/data-plane/Azure.Analytics.Purview.Account
/data-plane/Azure.Analytics.Purview.Catalog
/data-plane/Azure.Analytics.Purview.DataMap
/data-plane/Azure.Analytics.Purview.DevopsPolicies
/data-plane/Azure.Analytics.Purview.MetadataPolicies
/data-plane/Azure.Analytics.Purview.PDS
/data-plane/Azure.Analytics.Purview.Scanning
/data-plane/Azure.Analytics.Purview.SelfServicePolicies
/data-plane/Azure.Analytics.Purview.Share
/data-plane/Azure.Analytics.Purview.Workflow
/data-plane/Azure.Batch
/data-plane/Azure.ContainerRegistry
/data-plane/Azure.Contoso.WidgetManager
/data-plane/Azure.Developer.Signing
/data-plane/Azure.ProgrammableConnectivity
/data-plane/Azure.Search
/data-plane/Azure.Storage.Files.DataLake
/data-plane/AzureOpenAI
/data-plane/CallAutomation
/data-plane/CallingServer
/data-plane/Chat
/data-plane/Common
/data-plane/ComputerVision
/data-plane/ContentModerator
/data-plane/ContentSafety
/data-plane/Creator
/data-plane/CustomImageSearch
/data-plane/CustomVision
/data-plane/CustomWebSearch
/data-plane/DEM
/data-plane/Data
/data-plane/DataRegistry
/data-plane/DocumentIntelligence
/data-plane/Email
/data-plane/EntitySearch
/data-plane/Face
/data-plane/FormRecognizer
/data-plane/Geolocation
/data-plane/HealthInsights
/data-plane/Identity
/data-plane/ImageAnalysis
/data-plane/ImageSearch
/data-plane/JobRouter
/data-plane/LUIS
/data-plane/Language
/data-plane/LiveMetrics
/data-plane/LocalSearch
/data-plane/MediaComposition
/data-plane/Messages
/data-plane/MetricsAdvisor
/data-plane/Microsoft.AVS
/data-plane/Microsoft.AgFoodPlatform
/data-plane/Microsoft.ApiCenter
/data-plane/Microsoft.ApiManagement
/data-plane/Microsoft.AppConfiguration
/data-plane/Microsoft.Attestation
/data-plane/Microsoft.Azure.Search.Data
/data-plane/Microsoft.Azure.Search.Service
/data-plane/Microsoft.Batch
/data-plane/Microsoft.BlobStorage
/data-plane/Microsoft.Cache
/data-plane/Microsoft.CodeTransparency
/data-plane/Microsoft.Communication
/data-plane/Microsoft.ConfidentialLedger
/data-plane/Microsoft.ContainerRegistry
/data-plane/Microsoft.ContainerService
/data-plane/Microsoft.DataBox
/data-plane/Microsoft.DataLakeAnalytics
/data-plane/Microsoft.DataLakeStore
/data-plane/Microsoft.DevCenter
/data-plane/Microsoft.DeviceUpdate
/data-plane/Microsoft.Devices
/data-plane/Microsoft.DigitalTwins
/data-plane/Microsoft.Easm
/data-plane/Microsoft.EventGrid
/data-plane/Microsoft.EventHub
/data-plane/Microsoft.FileStorage
/data-plane/Microsoft.GraphRbac
/data-plane/Microsoft.HDInsight
/data-plane/Microsoft.HealthcareApis
/data-plane/Microsoft.Insights
/data-plane/Microsoft.InstanceMetadataService
/data-plane/Microsoft.IoTCentral
/data-plane/Microsoft.KeyVault
/data-plane/Microsoft.LoadTestService
/data-plane/Microsoft.MachineLearningServices
/data-plane/Microsoft.ManagedCcf
/data-plane/Microsoft.Maps
/data-plane/Microsoft.Marketplace
/data-plane/Microsoft.Media
/data-plane/Microsoft.MixedReality
/data-plane/Microsoft.OperationalInsights
/data-plane/Microsoft.PlaywrightTesting.AuthManager
/data-plane/Microsoft.PolicyInsights
/data-plane/Microsoft.Quantum
/data-plane/Microsoft.QueueStorage
/data-plane/Microsoft.ResourceNotifications
/data-plane/Microsoft.Resources
/data-plane/Microsoft.SchemaRegistry
/data-plane/Microsoft.SecurityInsights
/data-plane/Microsoft.ServiceBus
/data-plane/Microsoft.ServiceFabric
/data-plane/Microsoft.SignalRService
/data-plane/Microsoft.Storage
/data-plane/Microsoft.StorageDataLake
/data-plane/Microsoft.Synapse
/data-plane/Microsoft.Tables
/data-plane/Microsoft.TimeSeriesInsights
/data-plane/Microsoft.VoiceServices
/data-plane/Microsoft.Web
/data-plane/Monitor.Exporters
/data-plane/NetworkTraversal
/data-plane/NewsSearch
/data-plane/OpenAI.Assistants
/data-plane/OperatorConnect
/data-plane/Personalizer
/data-plane/PhoneNumbers
/data-plane/Products
/data-plane/QnAMaker
/data-plane/Render
/data-plane/Reviews
/data-plane/Rooms
/data-plane/Route
/data-plane/Search
/data-plane/SipRouting
/data-plane/Sms
/data-plane/Spatial
/data-plane/SpeakerRecognition
/data-plane/Speech
/data-plane/SpellCheck
/data-plane/TextAnalytics
/data-plane/Timezone
/data-plane/Traffic
/data-plane/TranslatorText
/data-plane/UnifiedVision
/data-plane/VideoAnalyzer.Edge
/data-plane/VideoSearch
/data-plane/VisualSearch
/data-plane/Weather
/data-plane/WebPubSub
/data-plane/WebSearch
/data-plane/ingestion
/data-plane/preview
/data-plane/v1
```

Note the `/data-plane/preview`. There is one instance of it, in [monitor](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/monitor/data-plane/preview)

Reply:
I can't tell from the list above is a single "service team" has 2+ data-plane subdirs with README.md files. This would be useful to know to see if we have to support "service groups" for data-plane. Maybe these subdirs are the equivalent of "services" within a "service group" for data-plane.

Does anything actually use this subdir name for anything? If anything did, it would be an SDK and I do not think they use it. And, if they did, it is probably only the .NET SDK.
Can we find out if any tool actually uses this subdir name for anything?